deploy/cr.yaml

```diff
@@ -498,13 +498,6 @@ spec:
       enabled: false
       storageName: STORAGE-NAME-HERE
       timeBetweenUploads: 60
-#      resources:
```

Comment:
Why do you remove it?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1389/commits/f64244ce5936f578c154921949bece358fc51f32